Gruntfile.js

```diff
@@ -137,11 +140,29 @@ module.exports = function(grunt) {
 					'lib/commons/index.js',
 					'lib/commons/*/index.js',
 					'lib/commons/**/*.js',
+
+					// directories we've converted to ES Modules
+					'!lib/commons/utils/*.js',
+
+					// output of webpack directories
+					'tmp/commons/**/*.js',
+
 					'lib/commons/outro.stub'
 				],
 				dest: 'tmp/commons.js'
 			}
 		},
+		webpack: {
+			commonsUtils: {
+				devtool: 'source-map',
+				mode: 'development',
+				entry: path.resolve(__dirname, 'lib/commons/utils/index.js'),
```

Comment:
Since this file contains ES6+, I believe we'll want to run it thru Babel, but I don't see any custom loaders added here. How does this work?

Reply:
For now this will just concat things together. The build pipeline already babels so we don't want to do it twice. Once this is all converted over we can add the babel load to our webpack config.